frontend/templates/product/sections/ProductSection/ProductOptions.tsx

```diff
@@ -291,14 +277,12 @@ function ProductOptionImage({ image, exactMatch }: ProductOptionImageProps) {
          mb="1"
          opacity={exactMatch ? 1 : 0.4}
       >
-         <Img
-            w="auto"
-            maxH="full"
-            maxW="full"
-            mx="auto"
+         <ResponsiveImage
             src={image.url}
             alt=""
+            layout="fill"
```

Comment:
I wrote down some consideration regarding how srcSet are generated [here](https://github.com/iFixit/react-commerce/issues/1039#issuecomment-1330443421)

For this image we are saying that we want to fill a container whose height is 64px with an image with a suggested width of 64px. So this is some sort of fixed size image.

If instead of declaring it responsive we remove `layout=fill` and `sizes` and we provide `width` and `height`, `srcSet` will only contain pixel density specific variation instead of the whole set of `imageSizes` + `deviceSizes`

Reply:
Thanks for enlightening me. I took your advice.